tools/wavetool.py

```diff
@@ -0,0 +1,157 @@
+#! /usr/bin/python3
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+import os
+import sys
+import argparse
+import numpy as np
+import scipy.signal as signal
+import scipy.io.wavfile as wavefile
+import scipy.fftpack as fftpack
+
+eps = 0.0000000001 # used to avoid zero division error
+
+def generateSinusoid(**params):
```

Comment:
In a function so short you can rename `params` to just `p`, it will fix the problem of too long lines and focus the reader's attention on what matters: parameter names like 'freq'.
You can add a pydoc if you want to explain what `p` is.

Reply:
thanks.